<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CPoiMapEditDialog</class>
 <widget class="QMainWindow" name="CPoiMapEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>274</width>
    <height>308</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit POI to World Mappings</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QListView" name="ListView">
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="verticalScrollMode">
       <enum>QAbstractItemView::ScrollPerPixel</enum>
      </property>
      <property name="horizontalScrollMode">
       <enum>QAbstractItemView::ScrollPerPixel</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="AddMeshButton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../Icons.qrc">
          <normaloff>:/icons/Plus.png</normaloff>:/icons/Plus.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="RemoveMeshButton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../Icons.qrc">
          <normaloff>:/icons/Minus v2.png</normaloff>:/icons/Minus v2.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="ButtonBox">
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::Save</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="ToolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="ActionHighlightSelected"/>
   <addaction name="ActionHighlightAll"/>
   <addaction name="ActionHighlightNone"/>
  </widget>
  <action name="ActionHighlightAll">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Highlight All</string>
   </property>
  </action>
  <action name="ActionHighlightNone">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Highlight None</string>
   </property>
   <property name="toolTip">
    <string>Highlight None</string>
   </property>
  </action>
  <action name="ActionHighlightSelected">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Highlight Selected</string>
   </property>
   <property name="toolTip">
    <string>Highlight Selected</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../Icons.qrc"/>
 </resources>
 <connections/>
</ui>
